packages/config/src/projects/layer2s/telos.ts

```diff
@@ -0,0 +1,28 @@
+import { upcomingL2 } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const telos: Layer2 = upcomingL2({
+  id: 'telos',
+  display: {
+    name: 'Telos',
+    slug: 'telos',
+    description:
+      'Telos L2 will leverage the SNARKtor decentralized protocol for scalable SNARKs verification unlocking massively scalable users data protecting applications, regulator compliance and trustless interoperability.',
```

Comment:
pls change to 'Telos zkEVM is an upcoming L2 ZK-based scaling solution on Ethereum, powered by SNARKtor decentralized proof aggregation and developed by Telos' or something similar